client/ayon_blender/plugins/publish/extract_usd.py

```diff
@@ -80,6 +104,75 @@ def process(self, instance):
         instance.data.setdefault("representations", []).append(representation)
         self.log.debug("Extracted instance '%s' to: %s",
                        instance.name, representation)
+    
+    @classmethod
+    def get_attr_defs_for_instance(cls, create_context, instance):
+        # Filtering of instance, if needed, can be customized
+        if not cls.instance_matches_plugin_families(instance):
+            return []
+        
+        # Attributes logic
+        publish_attributes = instance["publish_attributes"].get(
+            cls.__name__, {})
```

Comment:
I wonder if `self.get_attr_values_from_data` could be used here? @iLLiCiTiT thoughts?